packages/service-library/src/servicelib/utils.py

```diff
@@ -79,8 +78,11 @@ def log_exception_callback(fut: asyncio.Future):
 
 # // tasks
 async def logged_gather(
-    *tasks, reraise: bool = True, log: logging.Logger = logger, max_concurrency: int = 0
-) -> List[Any]:
+    *tasks: Awaitable[Any],
+    reraise: bool = True,
+    log: logging.Logger = logger,
+    max_concurrency: int = 0,
+) -> List[Optional[Any]]:
```

Comment:
MINOR: does not ``Any`` include ``None``, i.e. ``List[Any]`` would suffice?

Reply:
I think mypy suggested this. [This](https://github.com/python/mypy/issues/3138) might provide further info.